protoc-gen-restate/build.gradle.kts

```diff
@@ -7,17 +7,17 @@ plugins {
   `library-publishing-conventions`
 }
 
-description = "Protoc plugin to generate interfaces compatible with dev.restate:sdk-java-blocking"
+description = "Protoc plugin to generate interfaces compatible with dev.restate:sdk-api"
 
 dependencies {
   compileOnly(coreLibs.javax.annotation.api)
   implementation("com.salesforce.servicelibs:jprotoc:1.2.2") {
     exclude("javax.annotation", "javax.annotation-api")
   }
-  implementation(project(":sdk-core"))
+  implementation(project(":sdk-api-common"))
```

Comment:
Could we rename `sdk-api-common` into `sdk-common`?

Reply:
Makes sense, it's also consistent with the package name.